app/controls/form/form.js

```diff
@@ -179,6 +179,12 @@ define([
                 'maxlength'     : null
             }, this.options, $el.data());
 
+            // Set the placeholder
+            var placeholder_text = $el.attr('placeholder');
```

Comment:
I couldn't find any mention of a `data-placeholder` in textarea.ejs only of a `placeholder` attribute. This does fix the issue - as previously placeholder text wasn't displaying from the fixture data.. and now it is.